code/addons/test/src/postinstall.ts

```diff
@@ -11,33 +11,37 @@ import {
   loadMainConfig,
   validateFrameworkName,
 } from 'storybook/internal/common';
-import { logger } from 'storybook/internal/node-logger';
+import { colors, logger } from 'storybook/internal/node-logger';
 
 import { findUp } from 'find-up';
+import { satisfies } from 'semver';
 import c from 'tinyrainbow';
 import dedent from 'ts-dedent';
 
 import { type PostinstallOptions } from '../../../lib/cli-storybook/src/add';
+import { printError, printInfo, printSuccess, step } from './postinstall-logger';
 
+const addonName = '@storybook/experimental-addon-vitest';
+const dependencies = ['vitest', '@vitest/browser', 'playwright'];
+const optionalDependencies = ['@vitest/coverage-istanbul', '@vitest/coverage-v8'];
 const extensions = ['.js', '.jsx', '.ts', '.tsx', '.cts', '.mts', '.cjs', '.mjs'];
```

Comment:
nit, because constants
```suggestion
const ADDON_NAME = '@storybook/experimental-addon-vitest' as const;
const DEPENDENCIES = ['vitest', '@vitest/browser', 'playwright'] as const;
const OPTIONAL_DEPENDENCIES = ['@vitest/coverage-istanbul', '@vitest/coverage-v8'] as const;
const EXTENSIONS = ['.js', '.jsx', '.ts', '.tsx', '.cts', '.mts', '.cjs', '.mjs'] as const;
```

Reply:
Although it looks like further down we're mutating the `dependencies` array? So maybe I'm wrong on that.

Reply:
👍